code/modules/admin/verbs/striketeam.dm

```diff
@@ -88,117 +105,86 @@ GLOBAL_VAR_INIT(sent_strike_team, 0)
+		P.name = "Special Operations Manual"
+		P.update_icon()
 		var/obj/item/stamp/centcom/stamp = new
 		P.stamp(stamp)
 		qdel(stamp)
 
-	for(var/thing in GLOB.landmarks_list)
-		var/obj/effect/landmark/L = thing
-		if(L.name == "Commando-Bomb")
-			new /obj/effect/spawner/newbomb/timer/syndicate(L.loc)
-			qdel(L)
-
-	message_admins("<span class='notice'>[key_name_admin(usr)] has spawned a CentComm DeathSquad.</span>", 1)
-	log_admin("[key_name(usr)] used Spawn Death Squad.")
-	return 1
+	message_admins("<span class='notice'>[key_name_admin(proccaller)] has spawned a DeathSquad.</span>")
+	log_admin("[key_name(proccaller)] used Spawn Death Squad.")
+	return TRUE
 
 /client/proc/create_death_commando(obj/spawn_location, is_leader = FALSE)
 	var/mob/living/carbon/human/new_commando = new(spawn_location.loc)
 	var/commando_leader_rank = pick("Lieutenant", "Captain", "Major")
 	var/commando_name = pick(GLOB.commando_names)
+	var/obj/item/organ/external/head/head_organ = new_commando.get_organ("head") // This appearance code is brought to you by ert.dm, basically the same code. If you change something here change somethere there too.
 
-	var/datum/character_save/S = new //Randomize appearance for the commando.
-	S.randomise()
 	if(is_leader)
-		S.age = rand(35, 45)
-		S.real_name = "[commando_leader_rank] [commando_name]"
+		new_commando.age = rand(35, 45)
+		new_commando.real_name = "[commando_leader_rank] [commando_name]"
 	else
-		S.real_name = "[commando_name]"
-	S.copy_to(new_commando)
+		new_commando.real_name = "[commando_name]"
 
-
-	new_commando.dna.ready_dna(new_commando)//Creates DNA.
+	if(prob(50))
+		new_commando.change_gender(MALE)
+	else
+		new_commando.change_gender(FEMALE)
+
+	new_commando.set_species(/datum/species/human, TRUE) // All of this code down here too is also from ert.dm, I'm lazy don't blame me
```

Comment:
Since you're commenting the block of code, rather than a single line
```suggestion
	// All of this code down here too is also from ert.dm, I'm lazy don't blame me
	new_commando.set_species(/datum/species/human, TRUE)
```